### PR TITLE
Added documentation for extraData option.

### PR DESCRIPTION
**Pull request by EndrII**
Added documentation for extraData option.

### CODE REVIEW

**Review by usermeme — CHANGES_REQUESTED** [review]

File: docs/en/ExtraFiles.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# Adding extraData into distributive.
+
+CQtDeployer support work with extra data files.
```
[the remark this answers is not in the document]
Comment (EndrII) [reply docs/en/ExtraFiles.md]: fixed

File: docs/en/ExtraFiles.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+```json
+{
+    ...
+    "extraData": "./path/to/my/customFile"
+    ...
+}
+```
+
+This command support list of arrguments.
```
Comment (usermeme) [inline docs/en/ExtraFiles.md:25]: arguments

File: docs/en/ExtraFiles.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        "./path/to/my/customFile2",
+        "./path/to/my/customFile3"
+    ]
+    ...
+}
+```
+
+### Changing location of extra data files.
+
+For change location you should use  the extraDataOut options. This option change destination path relative the root of the distributive.
```
Comment (usermeme) [inline docs/en/ExtraFiles.md:48]: This option change destination path based on the root of the distributive

File: docs/en/ExtraFiles.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+    ],
+    "extraDataOut": "scripts"
+    ...
+}
+```
+
+## Packages
+extraData and extraDataOut options have support to work with packages. See the packaging [page](Pacakges.md) to get more information about packages.
+
+Sometimes we need to create a multipackages application. For example create a installer for clinet-server application with the client and server applications. The client and server have a own default configuration files. We add cinfig files using the extraData option.
```
Comment (usermeme) [inline docs/en/ExtraFiles.md:73]: client-server

Comment (usermeme) [inline docs/en/ExtraFiles.md:73]: change cinfig to config

File: docs/en/ExtraFiles.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+For create a big distributon i recomends use the [confFile](DeployConfigFile.md).
```
Comment (usermeme) [inline docs/en/ExtraFiles.md:75]: change distributon to distribution. And word I should be in capital case

Comment (usermeme) [inline docs/en/ExtraFiles.md:75]: change recomens to "recommend", without 's' in the end

File: docs/ru/ExtraFiles.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
+# Добавление дополнительных данных в дистрибутив.
+
+CQtDeployer поддерживает работу с дополнительными файлами данных.
+
+Для этого используйте параметры -extraData и -extraDataOut.
+
+## Примеры:
+
+### Добавление собственныйх файлов в корневой каталог дистрибутива.
```
Comment (usermeme) [inline docs/ru/ExtraFiles.md:9]: собственных

File: docs/ru/ExtraFiles.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+    ],
+    "extraDataOut": "скрипты"
+    ...
+}
+```
+
+## Пакеты
+Опции extraData и extraDataOut поддерживают работу с пакетами. Дополнительную информацию о пакетах см. На [странице](Pacakges.md) упаковки.
+
+Иногда нам нужно создать многопакетное приложение. Например, создайте установщик для приложения Clinet-server с клиентскими и серверными приложениями. Клиент и сервер имеют собственные файлы конфигурации по умолчанию. Мы добавляем файлы cinfig с помощью опции extraData.
```
Comment (usermeme) [inline docs/ru/ExtraFiles.md:73]: client-server

Comment (usermeme) [inline docs/ru/ExtraFiles.md:73]: config

**Review by usermeme — CHANGES_REQUESTED** [review]

File: docs/en/ExtraFiles.md (hunk 3 above)
Comment (EndrII) [reply docs/en/ExtraFiles.md:48]: Why do you change relative to base? it changes the meaning.

**Pull request merged**